airflow/bin/cli.py

```diff
@@ -327,6 +326,21 @@ def run(args, dag=None):
         settings.configure_vars()
         settings.configure_orm()
 
+    if not args.pickle and not dag:
```

Comment:
@saguziel Just wondering why checking `is not None` is better? Isn't it redundant in this case?